DEVELOPMENT.rst

```diff
@@ -107,32 +107,91 @@ All other commands are the same as for GNU/Linux and Mac OS X.
 a. Running Angular.js test cases
 ''''''''''''''''''''''''''''''''
 
+Run them by starting karma::
+
     $ node_modules/.bin/karma start tests/karma/karma.conf.js
 
 
 Installation OpenSlides in big mode
 ===================================
 
-1. Install PostgreSQL und redis:
+1. Install PostgreSQL and redis
+-------------------------------
+
+    Just apt-get these packets::
+
+    $ sudo apt-get install postgresql redis-server libpq-dev --yes
+
+2. Configure postgresql
+-----------------------
+
+    * Add the postgres user to the system::
+
+        $ sudo adduser postgres --disabled-login --disabled-password
+
+    * Set the password for the postgres user::
+
+        $ sudo -u psql template1
+        ALTER USER postgres with encrypted password 'xxxxxx';
+        (Exit with Ctrl-D)
+
+    * Edit the connection method for the current user::
+
+        $ sudo vi /etc/postgresql/9.5/main/pg_hba.conf
+        change
+        local    all    postgres    peer
+        local    all    postgres    md5
+
+    * Restart postgres::
+
+        $ sudo /etc/init.d/postgresql restart
 
-apt-get install postgresql redis-server libpg-dev
+    * Create the database user. <username> should be the current user
+      or the user who will be running OpenSlides. You can get the current
+      user with 'whoami'::
 
-TODO: Configure postgresql
+        $ createuser -U postgres -d -e -E -l -P -r -s <username>
 
-2. Install python dependencies
+      The passwords questioned is first the new database user
+      password and last the postgres user password.
 
-pip install django-redis asgi-redis django-redis-sessions psycopg2
+    * Edit the connection method again::
+
+        $ sudo vim /etc/postgresql/9.5/main/pg_hba.conf
+        change
+        local    all    all    peer
+        local    all    all    md5
+
+    * Restart postgres again.
+
+    * Create openslides database::
+
+        $ createdb openslides
+
+2. Install python dependencies::
+
+    $ pip install -r requirements_production.txt
 
 3. Change settings.py
 
-(See comments in the settings)
+  (See comments in the settings)
+
+  The relevant settings are: DATABASES, use_redis, CHANNEL_LAYERS, CACHES
+
+4. If you created a fresh Database run this command to populate it::
+
+   $ python manage.py migrate
+
+5. Choose which protocol server do you want to use:
+
+  a) Daphne: Run one or more workers::
 
-The relevant settings are: DATABASES, CHANNEL_LAYERS, CACHES
+      $ python manage.py runworker [--threads 2]
 
-4. Start one or more workers:
+    Set the DJANGO_SETTINGS_MODULE and the PYTHONPATH to start daphne::
 
-python manage.py runworker
+      $ DJANGO_SETTINGS_MODULE=settings PYTHONPATH=personal_data/var/ daphne openslides.asgi:channel_layer
 
-5. Start daphne. Set the DJANGO_SETTINGS_MODULE and the PYTHONPATH
+  b) Geiss: Just run OpenSlides with this extra argument::
```

Comment:
In big mode we should not run geiss via openslides start but in extra process or terminal.